frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -183,6 +184,7 @@ export interface PipelineRun extends K8sResourceKind {
     // Odd status value that only appears in a single case - cancelling a pipeline
     status?: 'PipelineRunCancelled';
     timeout?: string;
+    finally?: PipelineTask[];
```

Comment:
This is new... but I don't see any issues with it.

Reply:
Removed it, I think I got confused.